pages/search-results/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Movie from "../../components/Movie";
+import BackButton from "../../components/BackButton";
+
+export default function SearchResultsPage() {
+  return (
+    <>
+      <BackButton />
+      <section>
+        <h2>Your search results:</h2>
+      </section>
+      <section>
+        <Movie />
+      </section>
```

Comment:
Do you really need this `<section>`'s?

```suggestion
        <h2>Your search results:</h2>
        <Movie />
```